vlingo-reactive-messaging-patterns/src/main/java/io/vlingo/reactive/messaging/patterns/recipientlist/MountainSuppliesOrderProcessor.java

```diff
@@ -0,0 +1,296 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+package io.vlingo.reactive.messaging.patterns.recipientlist;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import io.vlingo.actors.Actor;
+import io.vlingo.actors.testkit.TestUntil;
+import io.vlingo.reactive.messaging.patterns.recipientlist.PriceQuoteInterest.PriceQuote;
+import io.vlingo.reactive.messaging.patterns.recipientlist.RetailBasket.RetailItem;
+
+/**
+ * MountainSuppliesOrderProcessor maintains registry of @{@link QuoteProcessor} {@link Actor} instances
+ * interested in providing quotes according to constraints on total retail price of a basket of items. 
+ *
+ * @author brsg.io
```

Comment:
@alanStrait Thanks for providing Javadocs! I have a huge job to go retroactively provide Javadocs across the many repos that I felt were premature early on when things were in high flux. As the team has decided, we will have a contributor credits document associated with the platform where there is one place to see the platform contributors and ways that they contributed. Plus, you have sort of indelible credit via the github file header, whereas comments can be arbitrarily removed in the future. I understand that this is your company standard and not a conscious gesture to add this, so if you could be aware to remove the @author and @since info after your Javadocs comment is generated it'd be appreciated.